src/i18n/appConstants.js

```diff
@@ -0,0 +1 @@
+export const PROD_ZEEGUU_URL = "www.zeeguu.org";
```

Comment:
the only thing missing here is to change the RHS with: `process.env.REACT_APP_EXTENSION_ID or "www.zeeguu.org"? Then it uses the envvar if defined, and falls back on zeeguu.org otherwise. what do you think?

Reply:
I like this, I have changed the variable name and added a check to see if the environment variable is defined.